### ISSUE TITLE
Running mean and variance with `tf.keras.layers.Normalization`

### ISSUE THREAD

**Issue by markub3327** [keras-team/tf-keras#344]
**System information**.

TensorFlow version (you are using): 2.10.1
Are you willing to contribute it (Yes/No) : Yes

**Describe the feature and the current behavior/state**.

Can `tf.keras.layers.Normalization` be used in running mean and variance manner? 
I need adapt this layer as the first step in the custom training function during training.

Is effectively runs `adapt(data, batch_size=256, steps=1)` with 1 step on the whole batch at every `def train_step(self, data):`? 

Or better way is firstly runs `adapt(data, batch_size=256, steps=1)` before training and during training (in `def train_step(self, sample):`) runs only `update_state(data)` and then `finalize_state()`, what can have less Python overhead?

**Will this change the current api? How?**
Maybe no.

**Who will benefit from this feature?**
Reinforcement Learning

**[Contributing](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md)**

- Do you want to contribute a PR? (yes/no): no
- If yes, please read [this page](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md) for instructions
- Briefly describe your candidate solution(if contributing):
